packages/core/src/utilities.ts

```diff
@@ -63,3 +64,10 @@ export function applyStyles(
 
   return flattenedStyles;
 }
+
+export function getRealValue(value: string | number | undefined) {
```

Comment:
since this is only used for radio buttons (at least for now) do you mind calling this `getValueForRadioButton`? or `getValueForRadio)`?

Reply:
@peter done.